src/sqlfluff/core/dialects/base.py

```diff
@@ -263,9 +263,19 @@ def ref(self, name: str) -> ExpandedDialectElementType:
                     )
                 )
         else:  # pragma: no cover
+            if name.endswith("KeywordSegment"):
+                keyword_tip = (
+                    " Perhaps specify the keyword? "
+                    "https://github.com/sqlfluff/sqlfluff/wiki/Contributing-Dialect-Changes#keywords"  # noqa E501
+                )
+            else:
+                keyword_tip = ""
             raise RuntimeError(
```

Comment:
Would it be as simple as changing this `raise RuntimeError` to this:

```suggestion
            sys.exit(
```

Reply:
Just put away the computer (2 am here), but I think this looped the error and hung.

Reply:
OK going to merge this. We can revisit later.